app/views/layouts/application.html.erb

```diff
@@ -11,7 +11,6 @@
 
   <body class="govuk-template__body js-enabled">
     <%= render '/layouts/header' %>
-    <%= render '/layouts/phase_banner' %>
```

Comment:
I'm not sure why this has been deleted in this PR?

Reply:
Good catch @kyle, it shouldn't have been deleted.  Putting it back now